### PR TITLE
Feature/orm#3

### PR DESCRIPTION
**Pull request by senqpaupa**
# Improved BookingService Tests

## Changes Overview
Added comprehensive test suite for the `BookingService` with proper database handling and isolation.

## Key Improvements
- Configured tests to run on a separate test database
- Added database cleanup before and after each test
- Implemented proper test data setup and teardown
- Added thorough assertions for all booking operations

## Test Coverage
The test suite covers the following functionality:

1. **Booking Creation**
   - Validates all booking fields (name, email, phone, dates, guests)
   - Verifies total price calculation
   - Ensures proper database persistence

2. **House Bookings Retrieval**
   - Tests multiple bookings for a single house
   - Verifies correct booking collection

3. **House Availability Check**
   - Tests availability for free periods
   - Verifies overlapping booking detection
   - Validates edge cases for booking dates

4. **Booking Cancellation**
   - Verifies status change
   - Ensures persistence of cancelled status

## Technical Details
- Uses Symfony's `KernelTestCase` for integration testing
- Implements proper database transaction handling
- Uses TRUNCATE for clean database state between tests
- Maintains test isolation through proper entity manager handling

## Testing
To run the tests:
```bash
php bin/phpunit tests/Service/BookingServiceTest.php
```

## Notes
- All tests are independent and can be run in any order
- Database state is guaranteed to be clean for each test
- No external dependencies required beyond test database setup

### CODE REVIEW

**Comment by praisethedeviI** [comment]
PR смотрит не в ветку предыдущего PR'а, а в main

File: tests/Service/HouseServiceTest.php [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+        $this->entityManager = $kernel->getContainer()
+            ->get('doctrine')
+            ->getManager();
+            
+        $this->houseService = new HouseService($this->entityManager);
+    }
+
+    public function testCreateHouse(): void
+    {
+        // Создаем новый домик
```
Comment (Faner201) [inline tests/Service/HouseServiceTest.php:28]: Комментарии излишни
